WORKSPACE

```diff
@@ -67,12 +64,31 @@ bind(
     actual = "//tools:java_toolchain",
 )
 
+http_archive(
+    name = "rules_java",
+    url = "https://github.com/bazelbuild/rules_java/releases/download/0.1.1/rules_java-0.1.1.tar.gz",
+    sha256 = "220b87d8cfabd22d1c6d8e3cdb4249abd4c93dcc152e0667db061fb1b957ee68",
+)
+
 load("@rules_java//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
 
 rules_java_dependencies()
 
 rules_java_toolchains()
 
+http_archive(
+    name = "rules_proto",
+    sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
+    strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
+        "https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
```

Comment:
Why are both needed?

Reply:
Great question -- the instructions (https://github.com/bazelbuild/rules_proto#getting-started) had included them both and I didn't notice they were the same. I presume we'd rather use the second one and it works with just the one URL, second one removed

Reply:
I don't think they're the same--I think the idea is to have redundancy in case one fails. Preferring GitHub seems reasonable--if a GitHub mirror fails then we're going to have many more dependencies here also fail. :)